extension/src/support.c

```diff
@@ -16,6 +16,21 @@ PG_MODULE_MAGIC;
 #endif
 
 PG_FUNCTION_INFO_V1(make_call_subquery_support);
+PG_FUNCTION_INFO_V1(gapfill_delta_transition);
+PG_FUNCTION_INFO_V1(gapfill_increase_transition);
+PG_FUNCTION_INFO_V1(gapfill_rate_transition);
+PG_FUNCTION_INFO_V1(gapfill_delta_final);
+
+/* linker hack to make sure the rust code is actually linked in */
```

Comment:
why do we need this?

Reply:
also probably in wrong file

Reply:
postgres passes a flag to the linker asking it to garbage collect all unused functions which sees that our rust functions are never used by this library and deletes them. This makes the functions actually used so they stick around. There may be some linker-hackery I can do to mark the entire archive we pass in as used, but I want to be careful that we don't accidentally include too much.